roles/kubernetes/secrets/tasks/gen_certs_vault.yml

```diff
@@ -53,6 +53,12 @@
         {%- endif -%}
         {%- endfor -%}
         "127.0.0.1","::1","{{ kube_apiserver_ip }}"
+        {%- if supplementary_addresses_in_ssl_keys is defined -%}
```

Comment:
so that this yields a proper list, can you move L56-61 above line 55?
Also, add a comma (",") after "{{ ip_item }"